### DISCUSSION TITLE
Additional icons

### DISCUSSION THREAD

**Discussion by electricbolt** [GitbookIO/community#702] [Feature Requests]
Thanks for the brand new icons for pages in GitBook. Could I please request a few more icons in the programming language area. You already have Swift, Java, JavaScript icons. Could you add:

1. Kotlin (https://kotlinlang.org)
2. Dart (https://dart.dev/)
3. C (https://commons.wikimedia.org/wiki/File:C_Programming_Language.svg)
5. C++ (https://isocpp.org/std/the-committee)

Thanks.

**Comment by denisemcg** [comment]
I'd like to be able to add custom icons to the list. My product has several modules, each with a unique brand icon. For example, I'd like to add the brand icon for module X to all the pages relevant to module X. This would differentiate them from similar topics in product Y pages. This feature would help alleviate my issue with the lack of page tags/labels.
See also https://github.com/orgs/GitbookIO/discussions/648
